### ISSUE TITLE
UIS symbols with therion don't match standard UIS symbols

### ISSUE THREAD

**Issue by mlubej** [therion/therion#570]
Hi!

I noticed that some symbols in the Therion UIS symbol set don't match the official UIS symbol set.
- here is the official UIS symbol set: https://www.carto.net/neumann/caving/cave-symbols/uis_signatures_english.pdf
- here is the Therion UIS symbol set: https://therion.speleo.sk/wiki/_media/therion_symbols_20130407_.pdf

And, for example, if you look at the "chimney" line, we have:
|Therion|UIS|
|-|-|
|![image](https://github.com/therion/therion/assets/37869250/5cf759bb-cad9-4915-a70d-9aa56c508bdd)|![image](https://github.com/therion/therion/assets/37869250/c11148a3-1ea2-4eb1-9e9f-6b5b93db9b79)|

I was wondering if there are specific reasons to these differences?

**Comment by rsevero** [comment]
I believe that's a history leftover mixed with the lack of someone that takes the time to fix it.

**Comment by hbeni** [comment]
Fixed I think

**Comment by mbudaj** [comment]
I disagree with the swap, see the symbols 44 and 80 in https://www.arge-hoehle-stuttgart.de/bei22_mue.pdf

**Comment by hbeni** [comment]
I Agree. Then this was a fix somewhere in the past, and not an accident.
And mabe some misunderstanding. Because I definitvely have seen "Karren" with the "M" signature in cave plans of the Hirlatz, where I withnessed Karren myself, and not anostomosis. Maybe some local rite :)

I will revert my change, but keep the other one.
